packages/ui/Marquee/tests/Marquee-specs.js

```diff
@@ -147,8 +166,60 @@ describe('Marquee', () => {
 		expect(marquee).toHaveStyle({'--ui-marquee-spacing': expected});
 		done();
 	});
-});
 
+	test('should warn when marquees are nested', () => {
+		const spy = jest.spyOn(console, 'error').mockImplementation(() => {});
+		render(
+			<Marquee marqueeOn="render" marqueeOnRenderDelay={10} marqueeSpacing={80} >
+				<Marquee marqueeOn="render" marqueeOnRenderDelay={-10} marqueeSpacing={80} />
+			</Marquee>
+		);
+
+		act(() => jest.advanceTimersByTime(100));
+
+		const expected = 1;
+		const actual = spy.mock.calls.length;
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should not animate when marquee is disabled', () => {
+		render(<Marquee marqueeDisabled>{ltrText}</Marquee>);
+		const marquee = screen.getByText(ltrText).parentElement;
+
+		const expected = 'marquee';
+
+		expect(marquee).not.toHaveClass(expected);
+	});
+
+	test('should start marquee on focus if `marqueeOn` is focus', () => {
+		render(<Marquee marqueeOn="focus" marqueeDelay={10}>{ltrText}</Marquee>);
+		const marquee = screen.getByText(ltrText);
+
+		fireEvent.focus(marquee);
+
+		act(() => jest.advanceTimersByTime(100));
+
+		expect(marquee).toHaveStyle({'--ui-marquee-spacing': '50'});
+
+		// calling blur for code coverage purposes. onBlur does not trigger any visual changes in jsdom.
+		fireEvent.blur(marquee.parentElement.parentElement);
```

Comment:
I think we could add a test 'should stop marquee on blur if `marqueeOn` is focus` for testing blur handler. WDYT?

Reply:
Hm.. As you said, it looks like not triggering any visual changes whatsoever.